src/XMLSecEnc.php

```diff
@@ -315,7 +315,7 @@ public function encryptKey($srcKey, $rawKey, $append=true)
             $this->encKey = $encKey;
         }
         $encMethod = $encKey->appendChild($this->encdoc->createElementNS(self::XMLENCNS, 'xenc:EncryptionMethod'));
-        $encMethod->setAttribute('Algorithm', $srcKey->getAlgorith());
+        $encMethod->setAttribute('Algorithm', $srcKey->getAlgorithm());
```

Comment:
If this master will used to create a new 3.0 release train as is on the table, I suggest we remove the deprecated methods `XMLSecurityDSig::generate_GUID` and `XMLSecurityKey::getAlgorith` as well. This is not part of this PR yet.